### ISSUE TITLE
Ability to delete old episodes of specific TV show? 

### ISSUE THREAD

**Issue by Guruleenyc** [jorenn92/Maintainerr#1204]
Is it possible to create a rule to delete old (>7d) or watched episodes of specific TV show?

**Comment by ydkmlt84** [comment]
Yes. Come on over to the Discord for some more help. This is basic functionality of Maintainerr.
